code/__DEFINES/dcs/signals.dm

```diff
@@ -390,6 +390,14 @@
 #define COMSIG_LIVING_SHOCK_PREVENTED "living_shock_prevented"
 ///sent by stuff like stunbatons and tasers: ()
 #define COMSIG_LIVING_MINOR_SHOCK "living_minor_shock"
+///Sent from defibrillators when everything seems good and the user will be shocked: (defibber, defib_item, ghost)
+#define COMSIG_LIVING_BEFORE_DEFIB "living_pre_defib"
```

Comment:
Nitpick but I think this sounds better, and falls in line with other "pre" signals.
```suggestion
#define COMSIG_LIVING_PRE_DEFIB "living_pre_defib"
```